application-insights.tf

```diff
@@ -1,9 +1,17 @@
 resource "azurerm_application_insights" "appinsights" {
   name                = "${var.product}-${var.env}"
```

Comment:
Use join function to remove string interpolations, where applicable.

Reply:
Done the interpolation changes